packages/mds-db/tests/mds-db.spec.ts

```diff
@@ -19,16 +19,22 @@ import {
   DISTRICT_SEVEN,
   START_ONE_MONTH_AGO,
   POLICY_WITH_DUPE_RULE,
-  PUBLISHED_POLICY
+  PUBLISHED_POLICY,
+  PUBLISH_DATE_VALIDATION_JSON,
+  START_ONE_MONTH_FROM_NOW,
+  DELETEABLE_POLICY
 } from '@mds-core/mds-test-data'
-import { now, clone, NotFoundError, rangeRandomInt, uuid, ConflictError } from '@mds-core/mds-utils'
+import { now, clone, NotFoundError, rangeRandomInt, uuid, ConflictError, yesterday } from '@mds-core/mds-utils'
 import { isNullOrUndefined } from 'util'
 import MDSDBPostgres from '../index'
 import { dropTables, createTables, updateSchema } from '../migration'
 import { Trip } from '../types'
 import { configureClient, MDSPostgresClient, PGInfo } from '../sql-utils'
 
 const { env } = process
+const ACTIVE_POLICY_JSON = clone(POLICY_JSON)
+ACTIVE_POLICY_JSON.publish_date = yesterday()
+ACTIVE_POLICY_JSON.start_date = yesterday()
```

Comment:
You could do this by using a spread:

`const ACTIVE_POLICY_JSON = { ...POLICY_JSON, publish_date: yesterday(), start_date: yesterday() }`

Also applies to everywhere else we're using `.clone`